tests/utils/test_helpers.py

```diff
@@ -262,3 +263,27 @@ def assert_exactly_one(true=0, truthy=0, false=0, falsy=0):
     def test_exactly_one_should_fail(self):
         with pytest.raises(ValueError):
             exactly_one([True, False])
+
+    @pytest.mark.parametrize('mode', ['strict', 'truthy'])
+    def test_prune_dict(self, mode):
```

Comment:
The code in this test reads odd since you have one half configured via `@parameterize` and the other half in the if/else condition. Could you separate it into two tests, or add the expected dict to the `@parameterize` arguments so that you don't need the if/else condition in the test?

Reply:
true :)